debian/salsa-ci.yml

```diff
@@ -112,35 +120,64 @@ blhc:
   mysql --table -e "SELECT * FROM plugin;" mysql
   mysql --table -e "SHOW PLUGINS;" mysql
 
+# Readline was removed from Debian Sid (and Bullseye) in Feb 2021. To be able to install older
+# versions of MariaDB that depend on it, fetch and install it from Buster.
+.test-install-readline-in-sid-for-backwards-compat: &test-install-readline-in-sid-for-backwards-compat |
+  curl -sS -O http://ftp.de.debian.org/debian/pool/main/r/readline5/libreadline5_5.2+dfsg-3+b13_amd64.deb
+  apt-get -qq install --no-install-recommends --yes ./libreadline5_5.2+dfsg-3+b13_amd64.deb
+
 .test-enable-bullseye-repos: &test-enable-bullseye-repos
   # Replace any old repos with just Sid
   - echo 'deb http://deb.debian.org/debian bullseye main' > /etc/apt/sources.list
   # Upgrade minimal stack first
   - apt-get update -qq
   - apt-get install -y apt
 
-.test-enable-bullseye-backports-repos: &test-enable-bullseye-backports-repos |
```

Comment:
Why do you rename this to `buster` and then you do the exact opposite rename in the section below? Can't they just stay as-is?